src/components/Reports/AdditionalSalaryRequest/AdditionalSalaryRequest.tsx

```diff
@@ -20,19 +17,17 @@ const MainContent: React.FC = () => {
         <Typography variant="h5">{t('About this Form content')}</Typography>
       );
     case AdditionalSalaryRequestSectionEnum.CompleteForm:
-      return <SalaryRequestForm />;
+      return <CompleteForm />;
     case AdditionalSalaryRequestSectionEnum.Receipt:
       return <Typography variant="h5">{t('Receipt content')}</Typography>;
   }
 };
 
 export const AdditionalSalaryRequest: React.FC = () => {
   return (
-    <AdditionalSalaryRequestProvider>
-      <AdditionalSalaryRequestLayout
-        sectionListPanel={<SectionList />}
-        mainContent={<MainContent />}
-      />
-    </AdditionalSalaryRequestProvider>
+    <AdditionalSalaryRequestLayout
```

Comment:
This Layout component doesn't do much. What do you think about collapsing it into this component like I just did for the salary calculation?